src/rc/start-stop-daemon.c

```diff
@@ -1211,7 +1222,8 @@ start_stop_daemon(int argc, char **argv)
 			if ((strncmp(env->value, "RC_", 3) == 0 &&
 				strncmp(env->value, "RC_SERVICE=", 10) != 0 &&
 				strncmp(env->value, "RC_SVCNAME=", 10) != 0) ||
-			    strncmp(env->value, "SSD_NICELEVEL=", 14) == 0)
+				strncmp(env->value, "SSD_NICELEVEL=", 14) == 0 ||
```

Comment:
The style guide offers no guidance here. Should lines 1225 and 1226 be dedented by one step, making them line up with if clause that they're a part of? Or dedent by one step, and then indent with three spaces, making them line up with the open paren? on line 1222?